src/configuration/preprocessors/api/mapping-template.coffee

```diff
@@ -0,0 +1,27 @@
+module.exports = '''
+{
+  "url": {
+    "path": {
+      #foreach($param in $input.params().path.keySet())
+      "$param": "$util.escapeJavaScript($input.params().path.get($param))"
+      #if($foreach.hasNext),#end
+      #end
+    },
+    "query": {
+      #foreach($param in $input.params().querystring.keySet())
+      "$param": "$util.escapeJavaScript($input.params().querystring.get($param))"
+      #if($foreach.hasNext),#end
+      #end
+    }
+  },
+  "method": "$context.httpMethod",
+  "headers": {
+    #foreach($param in $input.params().header.keySet())
+    "$param": "$util.escapeJavaScript($input.params().header.get($param))"
+    #if($foreach.hasNext),#end
+    #end
+  },
+  "content" : $input.json('$')
+}
+'''
+
```

Comment:
I was planning to inject this Mapping Template string into the api.yaml.  Then I decided to start humble and just put the Mapping TEmplate in the yaml.